### ISSUE TITLE
ASCornerLayoutSpec not find？

### ISSUE THREAD

**Issue by CainLuo** [TextureCommunity/TextureSwiftSupport#69]
I didn't find the class ASCornerLayoutSpec, is it not added?

**Comment by muukii** [comment]
@CainLuo 
ah, it's not created for now since my project didn't use it.
but it also can be descriptor in TextureSwiftSupport.

**Comment by CainLuo** [comment]
@muukii 
So does TextureSwiftSupport have a corresponding LayoutSpec to achieve the layout in the figure? This is ASCellNode.

![image](https://user-images.githubusercontent.com/8857291/122748906-4bf70b00-d2bf-11eb-8919-dfa7ba579d6e.png)

**Comment by muukii** [comment]
@CainLuo 
yeah, I mean we can add a new layout descriptors that uses ASCornerLayoutSpec inside.
But, that layout you've pasted, which can be described with CenterLayout or RelativeLayout with using ZStack or overlay.
